src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationLoader.php

```diff
@@ -195,7 +198,20 @@ protected function loadFileConfiguration($configPath, $profile)
         }
 
         $basePath = rtrim(dirname($configPath), DIRECTORY_SEPARATOR);
-        $config = (array) Yaml::parse(file_get_contents($configPath));
+
+        if (
+            str_ends_with($configPath, '.php')
+            || str_ends_with($configPath, '.php.dist')
```

Comment:
We no longer support `'.php.dist'` files, this was removed in one of the iterations, so you should remove this

Reply:
@carlos-granados good catch, I even looked at that line and missed it. Another good reason for dropping that extension, makes this much cleaner.

Reply:
good catch indeed. I've fixed.